builtin/k8s/helm/platform.go

```diff
@@ -127,6 +127,8 @@ func (p *Platform) Deploy(
 		client.Description = ""
 		client.CreateNamespace = p.config.CreateNamespace
 
+		log.Info("install client: %#v", client)
```

Comment:
```suggestion
		log.Info("installing client: %#v", client)
```

Reply:
This feels debug level to me, not info. I'd also recommend that you pick the specific fields on `client` that you're interested in and log those, to avoid logging irrelevant data (or potentially logging a secret)

Lastly, the logger takes a sequence of key value pairs, not a string with args that are interpolated. Example: https://github.com/hashicorp/go-hclog#emit-an-info-level-message-with-2-keyvalue-pairs